src/app/ApplicationTemplate.Shared.Views/Configuration/LocalizationConfiguration.cs

```diff
@@ -109,10 +109,7 @@ public ResourceLoaderStringLocalizer(bool treatEmptyAsNotFound = true)
 
 	private LocalizedString GetLocalizedString(string name, params object[] arguments)
 	{
-		if (name is null)
-		{
-			throw new ArgumentNullException(nameof(name));
-		}
+		ArgumentNullException.ThrowIfNull(name);
```

Comment:
Is this going to display "name" or the value of name?

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.argumentnullexception.throwifnull?view=net-9.0#system-argumentnullexception-throwifnull(system-object-system-string)